src/SDKs/RecoveryServices.Backup/RecoveryServices.Backup.Tests/ScenarioTests/DpmTests.cs

```diff
@@ -0,0 +1,92 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Linq;
+using Microsoft.Azure.Management.RecoveryServices.Backup.Models;
+using Microsoft.Rest.ClientRuntime.Azure.TestFramework;
+using Xunit;
+
+namespace Microsoft.Azure.Management.RecoveryServices.Backup.Tests
+{
+
+    /// <summary>
+    /// To run this test case in record mode, we will have to do following presetup.
+    /// 1.	Create resource and resource group and replace the value accordingly in following lines of the code.
+    /// 			private const string dpmVaultName = "SDKTestDPMVault";
+    /// 			private const string dpmResourceGroupName = "SDKTestDPMRg";
+    ///
+    ///	2.	Setup a Venus Machine. [Minimum supported Venus - 11.0.137.0]    
+    ///	            - Venus setup will install MARS agent. [Minimum supported agent version - 2.0.9040.0]
+    ///	   Register with created RS vault.
+    ///	   Create a PG. Protect any Datasource from the dpm machine. 
+    ///             - In the below testcase, system state workload is protected from the venus machine. 
+    ///	    
+    ///	3.	Replace the values accordingly in TestSetting.json
+    ///				"DpmDataSourceName": "SystemState;System Protectiond52df287-f0eb-46db-9729-819878c5f132",
+    ///                     - DpmDataSourceName format is Workloadtype;DataSourceUniqueName  where DataSourceUniqueName is DataSourceName+DataSourceId        
+    ///	            "DpmProductionServerName": "Windows;VinoVenus.DPMDOM02.SELFHOST.CORP.MICROSOFT.COM",
+    ///	                    - DpmProductionServerName format is ContainerType;ContainerName
+    ///             "DpmBackupEngineName": "VinoVenus.DPMDOM02.SELFHOST.CORP.MICROSOFT.COM"
+    ///                     - DpmBackupEngineName format is BackupEngineName
+    /// </summary>
+    public class DpmTests : TestBase, IDisposable
+    {
+        private const string dpmVaultName = "SDKTestDPMVault";
+        private const string dpmResourceGroupName = "SDKTestDPMRg";
+
+        [Fact]
```

Comment:
@VinothJeyaraman please record all new tests using latest version of Resource Manager or latest version of dependent ARM sdk packages.
We had a lots of issue migrating majority of tests when we moved to VS2017, so we are asking everyone who are recording tests to record using latest SDK packages